.github/workflows/release-python-sdk.yml

```
@@ -122,6 +122,72 @@ jobs:
+          retention-days: 90
+
+      - name: Add SBOM to release assets
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            python-sdk/sbom-cyclonedx.json
+            python-sdk/sbom-cyclonedx.xml
+            python-sdk/sbom-summary.md
+          body_path: python-sdk/sbom-summary.md
+          append_body: true 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)